content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+### Nestia
+Automatic `SDK` and `Swagger` generator for the `NestJS`.
```

Comment:
Rather than a tagline here like you'd put in a README, consider writing out what this library solve. Take a look at some of the other pages:

Compodoc

> Compodoc is a documentation tool for Angular applications. Since Nest and Angular share similar project and code structures, Compodoc works with Nest applications as well.

Commander

> Expanding on the [standalone application](https://docs.nestjs.com/standalone-applications) docs there's also the [nest-commander](https://jmcdo29.github.io/nest-commander) package for writing command line applications in a structure similar to your typical Nest application

CRUD Generator

> Throughout the life span of a project, when we build new features, we often need to add new resources to our application. These resources typically require multiple, repetitive operations that we have to repeat each time we define a new resource